RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -343,55 +346,44 @@ private void configureButtonBindings() {
           () -> {
             driverClimbEnabledPressed = true;
             if (operatorClimbEnabledPressed && driverClimbEnabledPressed) {
-              climberEnabled = !climberEnabled;
-
               climberSubsystem.toggleClimberEnabled();
               rgbSubsystem.climberEnabled();
             }
           }
         )
         .whenReleased(() -> driverClimbEnabledPressed = false);
 
-      buttons.elevatorExtend.whenPressed(climberSubsystem::elevatorExtend);
+      buttons.elevatorExtend.whenPressed(climberSubsystem::manualElevatorExtend);
       buttons.elevatorExtend.whenReleased(climberSubsystem::elevatorStop);
 
-      buttons.elevatorRetract.whenPressed(climberSubsystem::elevatorRetract);
+      buttons.elevatorRetract.whenPressed(climberSubsystem::manualElevatorRetract);
       buttons.elevatorRetract.whenReleased(climberSubsystem::elevatorStop);
-
-      buttons.climbAuto.whenPressed(
-        () -> {
-          climberSubsystem.climbAuto();
-          rgbSubsystem.autoClimb();
-        }
-      );
+      
+      buttons.climbAuto.whenPressed(climberSubsystem::autoClimb);
+      buttons.climbAuto.whenReleased(climberSubsystem::autoClimbReleased);
+      buttons.resetClimbStuff.whenPressed(climberSubsystem::resetClimbStuff);
     }
-
-    //Shooter Buttons and Climber Buttons
-    if (config.enableClimberSubsystem || config.enableShooterSubsystem) {
-      buttons.hubShootOrFixedHookToggle.whenPressed(
-        () -> {
-          if (config.enableClimberSubsystem && climberEnabled) {
-            climberSubsystem.fixedHookToggler();
-          } else {
-            if (config.enableShooterSubsystem) {
-              shooterSubsystem.shootTop();
-              intakeSubsystem.ballManagementForward();
-            }
-          }
+    
+    //Shooter BUttons and Climber Buttons
+    if (config.enableShooterSubsystem)
+    {
+
+      buttons.hubShoot.whenPressed(() -> {
+        if (config.enableShooterSubsystem) {
+          shooterSubsystem.shootTop();
+          intakeSubsystem.ballManagementForward();
         }
-      );
-      buttons.hubShootOrFixedHookToggle.whenReleased(
-        () -> {
-          if (config.enableShooterSubsystem) {
-            shooterSubsystem.stopShoot();
-            intakeSubsystem.stopBallManagement();
-          }
+      });
+      buttons.hubShoot.whenReleased(() -> {
+        if (config.enableShooterSubsystem){
+          shooterSubsystem.stopShoot();
+          intakeSubsystem.stopBallManagement();
         }
-      );
-
+      });
+    
       buttons.tarmacShootOrToggleElevator.whenPressed(
         () -> {
-          if (config.enableClimberSubsystem && climberEnabled) {
+          if (config.enableClimberSubsystem && climberSubsystem.isClimberEnabled()) {
```

Comment:
The reason this was a boolean before is because if the climber is disabled, this throws an NPE because climberSubsystem is null

Reply:
actually, this should be fine. If the climberSubsystem is null, config. enableClimberSubsystem will be null